test/stores/photo.js

```diff
@@ -10,13 +10,13 @@
   See the License for the specific language governing permissions and
```

Comment:
More test accommodation for database assertions. Same comments on mocks apply here.